portable/NetworkInterface/xilinx_ultrascale/NetworkInterface.c

```diff
@@ -70,12 +70,21 @@
     #define PHY_LS_LOW_CHECK_TIME_MS    1000
 #endif
 
+#if ( ipconfigNETWORK_MTU > 1526 )
+    #warning the use of Jumbo Frames has not been tested sufficiently yet.
```

Comment:
I have tested your PR with an Ultrascale, connected through a switch to a Win 10 laptop.
I have created a command:
~~~
    jumbo send <length>
~~~
to send UDP messages of any length. My MTU is defined as 9000.

And as expected, the maximum payload size was 8972. Which leads to an Ethernet packet of 9014 bytes:
~~~c
Ethernet   14
IP         20
UDP         8
Payload  8972
-------------
         9014
~~~
It all looks good to me, and it works.

So if we leave in the \#warning about the use of jumbo frames, I approve this PR, and thank you for it.

Reply:
Thank you for testing. Interesting that it did not work for me for with payload size 8KB.
May also be a problem on my host side. I will test this a lot more in a few months.